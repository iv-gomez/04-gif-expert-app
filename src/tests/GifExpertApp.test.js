import React from "react";
const { shallow } = require("enzyme");
const { GifExpertApp } = require("../GifExpertApp");

describe("<GifExpertApp />", () => {
  test("should render ok", () => {
    const wrapper = shallow(<GifExpertApp />);
    expect(wrapper).toMatchSnapshot();
  });
});
