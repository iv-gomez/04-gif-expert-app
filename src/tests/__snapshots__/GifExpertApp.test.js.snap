// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GifExpertApp /> should render ok 1`] = `
<Fragment>
  <h2>
    GifExpertApp
  </h2>
  <AddCategory
    setCategories={[Function]}
  />
  <hr />
  <ol>
    <GifGrid
      category="one punch"
      key="one punch"
    />
  </ol>
</Fragment>
`;
